src/controllers/login.controller.js

```diff
@@ -0,0 +1,42 @@
+import Response from '../helpers/response';
+import UserHandler from '../services/login.services';
+import BcryptService from '../services/bcrypt.service';
+import AuthHandler from '../helpers/jwt';
+
+const response = new Response();
+/**
+ *
+ *
+ * @class loginHandler
+ */
+class loginHandler {
+  /**
+   *
+   *
+   * @static
+   * @param {req} req
+   * @param {res} res
+   * @returns {token} @memberof loginHandler
+   */
+  static async login(req, res) {
+    const user = await UserHandler.findUser(req.body);
+    const verifyPassword = BcryptService.verifyPassword(req.body.password, user.password);
+    if (!verifyPassword) {
```

Comment:
@victor-abz , you need to move validations to middleware files

Reply:
This is a controller. The validations are in their folder inside middleware folder.